third_party/terraform/tests/resource_google_project_iam_custom_role_test.go

```diff
@@ -27,6 +27,12 @@ func TestAccProjectIamCustomRole_basic(t *testing.T) {
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
+			{
+				ResourceName:      "google_project_iam_custom_role.foo",
+				ImportStateId:     roleId,
```

Comment:
In case you're unfamiliar, this option changes the value sent as the id in a simulated `terraform import`. So while the prior import does `terraform import projects/foo/roles/bar`, this will do `terraform import bar`

Reply:
Would it make sense to also do a third import with `ImportStateId: foo/bar` so that it's testing all three accepted formats?

Reply:
Yep!